doc/api/deprecations.md

```diff
@@ -2303,7 +2303,7 @@ with no performance impact since Node.js 10.
 ### DEP0XXX: ChildProcess._channel
```

Comment:
This should have a number. Raised https://github.com/nodejs/node/pull/27164 to fix.